README.md

```diff
@@ -12,20 +12,20 @@ Created by Andreas Friedrich, Luis de la Garza, Sven Fillinger.
 
 ## How to Install
 
-With Maven you can include the library as dependency with:
+With Maven you can include the recent library version as dependency with:
 
 ```XML
 <dependency>
   <groupId>life.qbic</groupId>
   <artifactId>data-model-lib</artifactId>
-  <version>1.12.0</version>
+  <version>2.8.1</version>
 </dependency>
 ```
 or Groovy Grape:
 
 ```
 @Grapes(
-  @Grab(group='life.qbic', module='data-model-lib', version='1.12.0')
+  @Grab(group='life.qbic', module='data-model-lib', version='2.8.1')
```

Comment:
Since this is the hotfix 2.8.2 this should also be the set to 2.8.2
```suggestion
  @Grab(group='life.qbic', module='data-model-lib', version='2.8.2')
```